README.md

```diff
@@ -57,26 +57,34 @@ Get-Module MicrosoftPowerBIMgmt* -ListAvailable | Uninstall-Module -Force
 > * Individual is used to access entities that belong to the current user.
 > * Organization is used to access entities across the entire company. Only Power BI tenant admins are allowed to use.
 
+If the `-Scope` parameter doesn't exist on the cmdlet, the entity doesn't support an Administrative API.
+
 ### Log in to Power BI
 
 ```powershell
-Connect-PowerBIServiceAccount   # or Login-PowerBIServiceAccount
+Connect-PowerBIServiceAccount   # or use aliases: Login-PowerBIServiceAccount, Login-PowerBI
 ```
 
 ### Get Workspaces
 
-Get workspaces for the user. By default (without -First parameter) it shows the first 100 workspaces assigned to the user.:
+Get workspaces for the user. By default (without `-First` parameter) it shows the first 100 workspaces assigned to the user:
 
 ```powershell
 Get-PowerBIWorkspace
 ```
 
-Introduced -All parameter to show all workspaces assigned to the user.
+With the `-All` parameter, all workspaces assigned to the user will be shown:
```

Comment:
"Use the `-All` parameter to show all workspaces assigned to the user"

Reply:
Fixed